### PR TITLE
ScenarioGUI refactor companion

### PR DESCRIPTION
**Pull request by orengg**
Companion pr for [source repo pr #219](https://github.com/cortex-command-community/Cortex-Command-Community-Project-Source/pull/219).

* Removes the quit confirm dialog.

* Pushes the back-to-main button to be always visible.

* Adds a GUI element to contain the scene preview bitmap.

### CODE REVIEW

**Review by garethyr — DISMISSED** [review]
Cool, I imagine this is fine. I'll merge it after the source branch is merged in.

**Review by garethyr — APPROVED** [review]
Seems fine

**Comment by MaximDude** [comment]
Merged into [Menus-Refactor](https://github.com/cortex-command-community/Cortex-Command-Community-Project-Data/tree/Menus-Refactor).

**Pull request closed without merging**
